packages/inline-banner/addon/templates/components/nucleus-inline-banner.hbs

```diff
@@ -1,24 +1,21 @@
-{{#if hasBlock}}
-  {{yield
-    (hash
-      close=(action "onCloseTip")
-    )
-  }}
-{{else}}
-  <div class="nucleus-inline-banner__icon">
-    {{nucleus-icon name=_icon size="medium" variant=type}}
+<div class="nucleus-inline-banner__icon">
+  {{nucleus-icon name=_icon size="medium" variant=type}}
+</div>
+<div class="nucleus-inline-banner__content">
+  {{#if hasBlock}}
+    {{yield}}
+  {{/if}}
+  <div data-test-id="nucleus-inline-banner-title">
```

Comment:
Shouldn't this be `else` part? Otherwise, it'd get rendered for hasBlock scenario too, which is not what we want right?

Reply:
I'll do this change..